lib/client.ts

```diff
@@ -65,10 +71,53 @@ export class Client {
       };
     }
 
-    const tokenData = await this.oauth2Client.code.getToken(
-      callbackUri,
-      options
+    const token = await this.oauth2Client.code.getToken(callbackUri, options);
+
+    this.token = token;
+
+    return token;
+  };
+
+  public setToken = (
+    accessToken: string,
+    refreshToken?: string,
+    tokenType?: string
+  ): ClientOAuth2.Token => {
+    const data = {};
+    let token;
+
+    if (tokenType && refreshToken) {
+      token = this.oauth2Client.createToken(
+        accessToken,
+        refreshToken,
+        tokenType,
+        data
+      );
+    } else if (refreshToken) {
+      token = this.oauth2Client.createToken(accessToken, refreshToken, data);
+    } else {
+      token = this.oauth2Client.createToken(accessToken, data);
+    }
+
+    this.token = token;
+    return token;
+  };
+
+  public rawQuery = async (
```

Comment:
I was wondering if it would be nice to have a separate helper for mutation 🤔but I'm not sure if it would be easy to do with this `graphql-request` module, I don't know it.

It's just a thought for the future anyway, for now we can do with just this one helper it will be totally fine